migrations/versions/0171_add_org_invite_template.py

```diff
@@ -72,11 +72,24 @@ def upgrade():
         )
     )
 
+# If you are copying this migration, please remember about an insert to TemplateRedacted,
+# which was not originally included here either by mistake or because it was before TemplateRedacted existed
+    # op.execute(
+    #     """
+    #         INSERT INTO template_redacted (template_id, redact_personalisation, updated_at, updated_by_id)
+    #         VALUES ('{}', '{}', '{}', '{}')
+    #         ;
+    #     """.format(template_id, False, datetime.utcnow(), current_app.config['NOTIFY_USER_ID'])
+    # )
+
     # clean up constraints on org_to_service - service_id-org_id constraint is redundant
     op.drop_constraint('organisation_to_service_service_id_organisation_id_key', 'organisation_to_service', type_='unique')
 
 
 def downgrade():
+    op.execute("DELETE FROM notifications WHERE template_id = '{}'".format(template_id))
+    op.execute("DELETE FROM notification_history WHERE template_id = '{}'".format(template_id))
     op.execute("DELETE FROM templates_history WHERE id = '{}'".format(template_id))
     op.execute("DELETE FROM templates WHERE id = '{}'".format(template_id))
+    op.execute("DELETE FROM template_redacted WHERE template_id = '{}'".format(template_id))
```

Comment:
FYI: If the delete from template_redacted is before templates it will fail because of the foreign key constraint

Reply:
ahh yes, I tested it now and I have to put statement deleting from template_redacted before statement deleting from templates 👍. I will do that now ☺️